test/extended/authentication/front_proxy.go

```diff
@@ -33,6 +33,14 @@ var _ = g.Describe("[sig-auth][Feature:Authentication] ", func() {
 			if *controlPlaneTopology == configv1.ExternalTopologyMode {
 				e2eskipper.Skipf("External clusters do not have an aggregator-client secret in the cluster. Because the control plane lives outside the cluster, the aggregator-client secret is not needed in the cluster.")
 			}
+
+			// Temporary workaround until https://issues.redhat.com/browse/OCPBUGS-17431 is fixed.
```

Comment:
don't refer to bugs in code or comments - add the reference in the PR description